kinto/__init__.py

```diff
@@ -44,44 +43,44 @@ def main(global_config, config=None, **settings):
         config = Configurator(settings=settings, root_factory=RouteFactory)
 
     # Force project name, since it determines settings prefix.
-    config.add_settings({'kinto.project_name': 'kinto'})
+    config.add_settings({"kinto.project_name": "kinto"})
 
-    kinto.core.initialize(config,
-                          version=__version__,
-                          default_settings=DEFAULT_SETTINGS)
+    kinto.core.initialize(
+        config, version=__version__, default_settings=DEFAULT_SETTINGS
+    )
 
     settings = config.get_settings()
 
     # Expose capability
-    schema_enabled = asbool(
-        settings['experimental_collection_schema_validation']
-    )
+    schema_enabled = asbool(settings["experimental_collection_schema_validation"])
     if schema_enabled:
         config.add_api_capability(
-            'schema',
-            description='Validates collection records with JSON schemas.',
-            url='https://kinto.readthedocs.io/en/latest/api/1.x/'
-                'collections.html#collection-json-schema')
+            "schema",
+            description="Validates collection records with JSON schemas.",
+            url="https://kinto.readthedocs.io/en/latest/api/1.x/"
+            "collections.html#collection-json-schema",
```

Comment:
«That must be a bug, that can't possibly be intentional» says my neighboor

Reply:
:sob: